src/test/java/pl/widulinski/webcrawlertool/ScrAppApplicationTests.java

```diff
@@ -0,0 +1,157 @@
+package pl.widulinski.webcrawlertool;
+
+import lombok.extern.slf4j.Slf4j;
+import org.apache.poi.openxml4j.exceptions.InvalidFormatException;
+import org.junit.jupiter.api.Test;
+import org.junit.runner.RunWith;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.web.servlet.MockMvc;
+import pl.widulinski.webcrawlertool.FoundWebElements.FoundWebElement;
+import pl.widulinski.webcrawlertool.createExcelFiles.CreateExcel;
+import pl.widulinski.webcrawlertool.enums.Categories;
+import pl.widulinski.webcrawlertool.searchData.PreparedDataToSrchDto;
+import pl.widulinski.webcrawlertool.webDataToScrap.DataToScrap;
+import pl.widulinski.webcrawlertool.webDataToScrap.DataToScrapRepository;
+
+import java.io.File;
+import java.io.IOException;
+import java.text.SimpleDateFormat;
+import java.util.*;
+import java.util.stream.Stream;
+
+import static org.hamcrest.Matchers.containsString;
+import static org.junit.Assert.assertTrue;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+
+
+@RunWith(SpringRunner.class)
+@AutoConfigureMockMvc
+@Slf4j
+@SpringBootTest
+class ScrAppApplicationTests {
+
+    @Autowired
+    CreateExcel createExcel;
+
+    @Autowired
+    private PreparedDataToSrchDto preparedDataToSrchDto;
```

Comment:
jak dto i klasy modelowe przestana być komponentami to ten autowire trzeba zastapic `new PreparedDataToSrchDto()`

Reply:
zrobione